pkg/printers/internalversion/printers.go

```diff
@@ -60,7 +59,20 @@ import (
 	"k8s.io/kubernetes/pkg/util/node"
 )
 
-const loadBalancerWidth = 16
+const (
+	loadBalancerWidth = 16
+
+	// LabelNodeRoleMaster specifies that a node is a master
+	// It's copied over to here until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
+	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
```

Comment:
I haven't been part of the discussion here, so I assume it has happened and I just need a catch-up.  why is this not a taint?  I don't like the idea of explicit roles, especially in a self-hosted world.  The "master" might be spread around the cluster.